hivemind/moe/server/expert_backend.py

```diff
@@ -1,9 +1,10 @@
-from typing import Any, Callable, Dict, Sequence, Tuple, Union
+from typing import Any, Dict, Optional, Sequence, Tuple, Union
 
 import torch
 from torch import nn
 
 from hivemind.moe.server.task_pool import TaskPool
+from hivemind.optim.state_averager import LRSchedulerBase
```

Comment:
Since this import is not actually related to hivemind.optim, I’d suggest so simply inline the statement that is being imported

Reply:
fixed